### ISSUE TITLE
Where is dzl_preferences_view_new?

### ISSUE THREAD

**Issue by ZanderBrown** [chergert/libdazzle#12]
@chergert `dzl_preferences_view_new` is listed in `dzl-preferences-view.h`, [here](https://people.gnome.org/~chergert/docs/libdazzle/DzlPreferencesView.html#dzl-preferences-view-new) and the generated GIR/VAPI

However there doesn't seem to be any implementation leading to a linker error.

Have checked this repo + symbols in generated so

**Comment by ZanderBrown** [comment]
Obviously the workaround is to call `g_object_new` or in Vala:

```vala
var view = GLib.Object.new(typeof(Dazzle.PreferencesView)) as Dazzle.PreferencesView;
```

**Comment by chergert** [comment]
Thanks! Fixed in commit cfcea3e526e54729571a3ae95a3cb7766b28925b
